src/components/DpButton/DpButton.vue

```diff
@@ -66,7 +66,7 @@ const props = defineProps({
   },
 
   /**
-   * When passing an href, a link is rendered instead of a button element.
+   * When passing a href, a link is rendered instead of a button element.
```

Comment:
A google search, as well as stressing Deepl, says that "a href" is the correct form.

Reply:
Unless `a` refers to the html element, which in this case I don't think it does, I'm convinced it has to be `an`:
> An href would be correct, because it's correctly pronounced as aitch-ref. Since this gives it the sound of a leading vowel, you use an.

https://english.stackexchange.com/questions/79175/a-href-or-an-href

Reply:
thanks for insisting here :) the reasoning in https://english.stackexchange.com/a/79176 nailed it for me